contents/docs/libraries/ios/usage.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: iOS SDK usage
+sidebarTitle: iOS
+sidebar: Docs
+showTitle: true
+github: 'https://github.com/PostHog/posthog-ios'
+icon: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/ios.svg
+features:
+  eventCapture: true
+  userIdentification: true
+  autoCapture: true
+  sessionRecording: true
+  featureFlags: true
+  groupAnalytics: true
+  surveys: true
+  llmAnalytics: false
+---
+
+import IOSIdentify from './_snippets/identify.mdx'
+import IOSSendEvents from '../../integrate/send-events/_snippets/send-events-ios.mdx'
+
+
+## Capturing events
+
+<IOSSendEvents />
```

Comment:
There's another `Autocapture configuration` section in here where we could link to the config doc.